Formula/u/urlscan.rb

```diff
@@ -0,0 +1,25 @@
+class Urlscan < Formula
+  include Language::Python::Virtualenv
+
+  desc "View/select the URLs in an email message or file"
+  homepage "https://github.com/firecat53/urlscan"
+  url "https://files.pythonhosted.org/packages/b7/7d/984994a32b261cc3a72d3bfb0b8c0de4f786682128dc659a5c5e02dfc48c/urlscan-1.0.1.tar.gz"
+  sha256 "e0ec986e5aa2da57dd2face8692116d80af173d4eb56a78e4fd881731113307f"
+  license "GPL-2.0-only"
```

Comment:
```suggestion
  license "GPL-2.0-or-later"
```

license header ref, https://github.com/firecat53/urlscan/blob/main/urlscan/urlscan.py#L1-L17